app/main.py

```diff
@@ -1 +1,32 @@
-# write your code here
+class Animal:
+    health = 100
+    name = ""
+    hidden = False
+    alive = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.name = name
+        self.health = health
+        self.alive.append(self)
+
+    def __repr__(self) -> None:
```

Comment:
```suggestion
    def __repr__(self) -> str:
```
ШІ може звернути увагу на невірну анотацію типу Output, повинно бути `str`